apps/web/src/services/iam/components/UserGroupChannelSetInputForm.vue

```diff
@@ -51,6 +53,25 @@ const { key: notificationProtocolQueryKey, params: notificationProtocolQueryPara
     })),
 });
 
+const { userGroupChannelAPI } = useUserGroupChannelApi();
+
+const { key: userGroupChannelGetQueryKey, params: userGroupChannelGetQueryParams } = useServiceQueryKey('alert-manager', 'user-group-channel', 'get', {
+    contextKey: computed(() => userGroupPageGetters.selectedUserGroupChannel?.[0]?.channel_id),
+    params: computed(() => ({
+        channel_id: userGroupPageGetters.selectedUserGroupChannel?.[0]?.channel_id ?? '',
+    })),
+});
+
+const { data: userGroupChannelData } = useScopedQuery({
+    queryKey: userGroupChannelGetQueryKey.value,
+    queryFn: () => userGroupChannelAPI.get(userGroupChannelGetQueryParams.value),
+    enabled: computed(() => !!userGroupPageGetters.selectedUserGroupChannel?.[0]?.channel_id),
+    initialData: () => ({
+        name: '',
+    }),
+    gcTime: 1000 * 60 * 2,
+}, ['DOMAIN', 'WORKSPACE']);
```

Comment:
suggestion: I recommend creating a custom hook (e.g., `use-user-group-channel-get-query`) to handle this.